### PR TITLE
Alert Notification doc

### PR DESCRIPTION
**Pull request by ranbena**
Documentation for v9 feature https://github.com/getredash/redash/pull/4170.

### CODE REVIEW

File: src/pages/kb/user-guide/alerts/alert-notification-template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+category: alerts
+parent_category: user-guide
+helpscout_url: https://help.redash.io/article/74-alert-notification-template
```
Comment (arikfr) [inline src/pages/kb/user-guide/alerts/alert-notification-template.md]: This is not needed.

File: src/pages/kb/user-guide/alerts/alert-notification-template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+keywords:
+- Alert
+- Notification
+- Template
+- Custom Template
+- Formatting
+- Email
+title: Alxert Notification Templating
```
Comment (arikfr) [inline src/pages/kb/user-guide/alerts/alert-notification-template.md]:
```suggestion
title: Alert Notification Templating
```

**Comment by susodapop** [comment]
>     1. I think it's quite the huge page already.

True.

> Any static content is valid, and you can also incorporate some built-in template variables:
>
>    ALERT_STATUS - The evaluated alert status (string)
    ALERT_CONDITION - The alert condition operator (string).
    ALERT_THRESHOLD - The alert threshold (string or number).
    ALERT_NAME - The alert name (string).
    ALERT_URL - The alert page url (string).
    QUERY_NAME - The correlated query name (string).
    QUERY_URL - The correlated query page url (string).
    QUERY_RESULT_VALUE - The query result value (string or number)
    QUERY_RESULT_ROWS - The query result rows (value array)
    QUERY_RESULT_COLS - The query result columns (string array),

But these variable names make the most sense in context of the "Setting Up An Alert" page. Specifically the `ALERT_STATUS` and `ALERT_CONDITION` variables. If nothing else, could we enumerate the possible values for both variables?

**Comment by ranbena** [comment]
How about adding an exact link to the relevant sections in the setup-alert page from the variable descriptions?

**Comment by ranbena** [comment]
Added inline links to config setting explanations.

Screen Shot 2019-10-02 at 9 26 58

**Comment by ranbena** [comment]
Gonna land this. Any change requests will be handled in new PR.

**Comment by arikfr** [comment]
I restored the branch so we can push this again.

**Pull request merged**